client/log/go-kit/logger_test.go

```diff
@@ -0,0 +1,36 @@
+package xkit
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/go-kit/kit/log"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestLogger_Infof(t *testing.T) {
+	buf := &bytes.Buffer{}
+	logger := NewLogger(log.NewLogfmtLogger(buf))
+
+	logger.Infof("Formatted %s", "string")
+
+	assert.Equal(t, "level=info msg=\"Formatted string\"\n", buf.String())
```

Comment:
nit: instead of escaping `\"` use backticks ` for the outer string

Reply:
Unfortunately this won't work because of the newline at the end of the string.